packages/@aws-cdk-testing/framework-integ/test/aws-ses/test/integ.dedicated-ip-pool.ts

```diff
@@ -0,0 +1,19 @@
+import { App, Stack } from 'aws-cdk-lib';
+import * as integ from '@aws-cdk/integ-tests-alpha';
+import * as ses from 'aws-cdk-lib/aws-ses';
+
+const app = new App();
+const stack = new Stack(app, 'aws-cdk-dedicated-ip-pool');
+
+// new ses.DedicatedIpPool(stack, 'Pool', {
+//     dedicatedIpPoolName: 'myPool',
+//     scalingMode: ses.ScalingMode.MANAGED,
+//   });
+
+new ses.DedicatedIpPool(stack, 'Pool', {});
```

Comment:
```suggestion
new ses.DedicatedIpPool(stack, 'Pool', {
  dedicatedIpPoolName: 'myPool',
  scalingMode: ses.ScalingMode.MANAGED,
});
```
Incomplete integration test.

Reply:
Thank you for your review!

I tried conducting an integration test, but failed to create an IP pool. It seems that the 'poolName' must be within 64 characters and only allow lowercase letters, numbers, periods, dashes and underscores. (Deployment fails because 'myPool' includes uppercase letters.)

I have added the above conditions to the description of 'dedicatedIpPoolName', but is it okay to include such a modification in this Pull Request?